src/providers/jspm.ts

```diff
@@ -181,8 +183,25 @@ async function lookupRange (this: Resolver, registry: string, name: string, rang
     case 200:
       return { registry, name, version: (await res.text()).trim() };
     case 404:
-      return null;
+      const versions = await fetchVersions(name)
+      const semverRange = new SemverRange(range || '*', unstable)
+      const version = semverRange.bestMatch(versions, unstable);
+
+      return { registry, name, version };
```

Comment:
I believe the condition check is still needed even with `bestMatch` since there might be no match.

Reply:
Yep, I resolved it and added a test case.